misk-aws/src/main/kotlin/misk/dynamodb/DynamoDbHelpers.kt

```diff
@@ -0,0 +1,42 @@
+package misk.dynamodb
+
+import com.amazonaws.services.dynamodbv2.model.AmazonDynamoDBException
+import java.time.Duration
+import misk.backoff.Backoff
+import misk.backoff.DontRetryException
+import misk.backoff.ExponentialBackoff
+import misk.backoff.retry
+
+object DynamoDbHelpers {
+  fun getDefaultBackoff(): ExponentialBackoff {
+    return ExponentialBackoff(
+        baseDelay = Duration.ofMillis(100),
+        maxDelay = Duration.ofSeconds(10),
+        jitter = Duration.ofMillis(200)
```

Comment:
These numbers might lead to connection timeout. All said, I am not sure how can we judge good numbers here. Maybe using default values from other places might make sense